extensions/integration-tests/src/schemaCompare.test.ts

```diff
@@ -57,24 +57,24 @@ class SchemaCompareTester {
 		const now = new Date();
 		const operationId = 'testOperationId_' + now.getTime().toString();
 
-			let source: azdata.SchemaCompareEndpointInfo = {
```

Comment:
this formatting didn't even show in my vscode.. let me know if you want me to revert it.